addons/onboarding/models/onboarding_onboarding.py

```diff
@@ -94,12 +94,12 @@ def _prepare_rendering_values(self):
         self.ensure_one()
         values = {
             'bg_image': f'/web/image/onboarding.onboarding/{self.id}/panel_background_image',
+            'classes': f'o_onboarding_{self.panel_background_color}'
+                       if self.panel_background_color != 'none' else '',
```

Comment:
We could add also a condition to check if `panel_background_color` is set or adding a `required="1"` in the view. As this field is not required and we could end up with a class `o_onboarding_False`. So I let you decide what you prefer :smile: